### DISCUSSION TITLE
API/Schedule "is_now" always false

### DISCUSSION THREAD

**Discussion by multiburst** [AzuraCast/AzuraCast#3942] [Q&A]
Hello there,

first of all, thanks for all your work - AzuraCast is really great!

One question: when using the API to get the schedule via
/station/{station_id}/schedule

none of the returned entries has the "is_now" bool set to true - allthough the show is currently running (and has been added already a while ago)....

Is there something special to consider?

Thanks for your help and all the best,

Micha

**Comment by BusterNeece** [comment]
This should be fixed as of the latest rolling release update.
